CarSharingApp.Web/Controllers/RegistrationController.cs

```diff
@@ -40,8 +41,13 @@ public async Task<IActionResult> Index()
         }
 
         [AllowAnonymous]
+        [ValidateAntiForgeryToken]
+        [PreventDuplicateRequest]
```

Comment:
add to Demo

Reply:
Added attributes to all POST methods in the project